projects/test-utils/package.json

```diff
@@ -3,11 +3,14 @@
   "private": true,
   "version": "0.0.1",
   "peerDependencies": {
-    "@angular/common": "^9.1.0",
-    "@angular/core": "^9.1.0",
-    "@angular/flex-layout": "^9.0.0-beta.29",
-    "@angular/platform-browser": "^9.1.0",
-    "@angular/router": "^9.1.0",
-    "rxjs": "~6.5.4"
+    "@angular/common": "^10.0.4",
+    "@angular/core": "^10.0.4",
+    "@angular/flex-layout": "^10.0.0-beta.32",
```

Comment:
I am surprised this one is still in beta

Reply:
https://github.com/angular/flex-layout/issues/835